File: crypto/vnx-crypto-006/weak_rsa.py
from cryptography.hazmat.primitives.asymmetric import rsa
from cryptography.hazmat.backends import default_backend

# VNX-CRYPTO-006: Weak RSA key size (1024 bits)
private_key = rsa.generate_private_key(
    public_exponent=65537,
    key_size=1024,
    backend=default_backend()
)

File: crypto/vnx-crypto-006/weak_rsa.go
package main

import (
	"crypto/rand"
	"crypto/rsa"
	"fmt"
)

// VNX-CRYPTO-006: Weak RSA key size (1024 bits)
func main() {
	key, _ := rsa.GenerateKey(rand.Reader, 1024)
	fmt.Println(key.PublicKey.N)
}
